src/components/BooksTap/BookLayout.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import SubHeader from '../../pages/Main/SubHeader/SubHeader';
+
+const BookLayout = props => (
```

Comment:
```suggestion
const BookLayout = ({ title }) => (
```
함수 컴포넌트 props 자리에서 구조분해 할당 할 수 있습니다

Reply:
구조분해 할 수 있는 부분들 확인하고 진행했습니다!! 감사합니다!